tasks/models.py

```diff
@@ -1,3 +1,39 @@
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Tag(models.Model):
+    tag_name = models.CharField(max_length=50, unique=True, primary_key=True)
+
+    def __str__(self):
+        return self.tag_name
+
+
+class Task(models.Model):
+    task_code = models.CharField(max_length=8, unique=True, primary_key=True)
+    title = models.CharField(max_length=255)
+    author = models.ForeignKey(User, on_delete=models.CASCADE, to_field="username")
+    description = models.TextField()
+    tags = models.ManyToManyField(Tag)
+    score_mode = models.CharField(max_length=10)
+    score_parameter = models.IntegerField(default=None)
```

Comment:
Should be a string, since if we want to support IOI-style scoring (subtasks) then score_parameter will be a JSON string.

Reply:
Ok agreed